ShopperInsights/src/main/java/com/braintreepayments/api/shopperinsights/ExperimentType.kt

```diff
@@ -0,0 +1,15 @@
+package com.braintreepayments.api.shopperinsights
+
+enum class ExperimentType(val rawValue: String) {
```

Comment:
Do we need a doc string for this enum? Can `rawValue` also be an internal property, or can it even be private if it's only used in `formattedExperiment()`?